karaf/init/src/main/java/org/apache/brooklyn/launcher/osgi/OsgiLauncher.java

```diff
@@ -199,4 +199,7 @@ public void setLocalBrooklynProperties(String localBrooklynProperties) {
         this.localBrooklynProperties = localBrooklynProperties;
     }
 
+    public void setDeserializingClassRenamesProvider(DeserializingClassRenamesProvider.OsgiConfigLoader val) {
```

Comment:
No, I'd remove the property, since it's a no-op in the Java code it's not needed. (ie. remove it from both the blueprint.xml and the Java code)